docker-compose.yml

```diff
@@ -62,7 +86,9 @@ services:
     build: .
     networks:
       - hmpps
-    depends_on: [redis]
+    depends_on:
+      - redis
+      - hmpps-accredited-programmes-api
```

Comment:
Why does this rely on `hmpps-accredited-programmes-api`?

Reply:
This was used to start the app with docker compose, so I wanted it to wait for the API to start so we could fetch courses, but we never actually use docker compose to start the app. I've removed this whole section now.